### DISCUSSION TITLE
Гуглобус и азимуты — делаем их общественно полезными

### DISCUSSION THREAD

**Discussion by iprst** [terratensor/svodd#113] [Идеи]
Что такое гуглобус и азимуты, как сделать их полезными обществу. 

Кратко первые три источника: 
[Тема о Бейруте](https://xn----8sba0bbi0cdm.xn--p1ai/qa/question/view-37951)
[Описания метода азимутов](https://cont.ws/@lockor/1793342)
[Обсуждение метода более подробно](https://xn----8sba0bbi0cdm.xn--p1ai/qa/question/view-38558)

О чём эта тема. 

В данный момент существует полу-автоматический метод получения данных и расчёта координат вероятных событий. Этот метод медленный, плохой костыль на коленке, хотя и оптимизированный за много месяцев использования. Его невозможно использовать публично, поскольку он слишком неоптимален — придирчив к форме входных данных, требует ручного управления при переходе между автоматическими процедурами, некоторые процедуры медленные, не существует публичного интерфейса для простых действий, и так далее. Требуется разработать некую концепцию использования этой методологии, поэтому нужно проанализировать её как в целом, так и на отдельных этапах, а также возможную интеграцию методологии в наш «поисковик». 

Предлагаю всех к обсуждению вопроса.

**Comment by audetv** [comment]
Поставил себе в план снова ознакомится с материалами. Читал ранее в разное время, указанные выше ссылки, надо освежить в памяти и вникнуть с целью поиска технических решений задачи.
Но уже сейчас знаю, что поддерживаю идею, и что будем делать.
